processor/src/test/java/org/mapstruct/ap/test/collection/forged/CollectionMappingTest.java

```diff
@@ -87,8 +87,8 @@ public void shouldForgeNewMapMappingMethod() {
             @Diagnostic(type = ErroneousCollectionNonMappableSetMapper.class,
                 kind = Kind.ERROR,
                 line = 17,
-                messageRegExp = "Can't map Collection element \".* nonMappableSet\" to \".* nonMappableSet\". "
-                    + "Consider to declare/implement a mapping method: .*."),
+                messageRegExp = "No target bean properties found: can't map Collection element \".* nonMappableSet\" "
```

Comment:
This should not have been changed. See my previous comment

Reply:
I'm not so sure.. the full error message (removing the wild card matches is:

`No target bean properties found: can't map Collection element "org.mapstruct.ap.test.collection.forged.Foo nonMappableSet" to "org.mapstruct.ap.test.collection.forged.Bar nonMappableSet". Consider to declare/implement a mapping method: "org.mapstruct.ap.test.collection.forged.Bar map(org.mapstruct.ap.test.collection.forged.Foo value)`

`Bar` is an empty bean (No properties).

Reply:
You are right, sorry for my mistake